app/services/ClaimToAdjustService.scala

```diff
@@ -120,6 +120,21 @@ class ClaimToAdjustService @Inject()(val financialDetailsConnector: FinancialDet
       }
     }
   }
+
+  def getAdjustPaymentOnAccountViewModel(nino: Nino)
```

Comment:
if you can rename this method to be aligned with the viewModel name, ie something like
getAmendablePoaViewModel or change viewModel name as it a bit confusing to read